content/en/observability_pipelines/setup/splunk.md

```diff
@@ -0,0 +1,345 @@
+{{% /tab %}}
+{{% tab "APT-based Linux" %}}
+No built-in support for load-balancing is provided, given the single-machine nature of the installation. You will need to provision your own load balancers using whatever your company's standard is.
+{{% /tab %}}
+{{% tab "RPM-based Linux" %}}
+No built-in support for load-balancing is provided, given the single-machine nature of the installation. You will need to provision your own load balancers using whatever your company's standard is.
+{{% /tab %}}
+{{< /tabs >}}
+
+### Buffering
+Observability Pipelines includes multiple buffering strategies that allow you to increase the resilience of your cluster to downstream faults. The provided sample configurations use disk buffers, the capacities of which are rated for approximately 10 minutes of data at 10Mbps/core for Observability Pipelines deployments. That is often enough time for transient issues to resolve themselves, or for incident responders to decide what needs to be done with the observability data.
+
+{{< tabs >}}
+{{% tab "AWS EKS" %}}
+For AWS, Datadog recommends using the `io2` EBS drive family. Alternatively, the `gp3` drives could also be used.
+{{% /tab %}}
+{{% tab "Azure AKS" %}}
+For Azure AKS, Datadog recommends using the `default` (also known as `managed-csi`) disks.
+{{% /tab %}}
+{{% tab "Google GKE" %}}
+For Google GKE, Datadog recommends using the `premium-rwo` drive class because it is backed by SSDs. The HDD-backed class, `standard-rwo`, might not provide enough write performance for the buffers to be useful.
+{{% /tab %}}
+{{% tab "APT-based Linux" %}}
+By default, the Observability Pipelines Worker's data directory is set to `/var/lib/observability-pipelines-worker` - if you are using the sample configuration, you should ensure that this has at least 288GB of space available for buffering.
+
+Where possible, it is recommended to have a separate SSD mounted at that location.
+{{% /tab %}}
+{{% tab "RPM-based Linux" %}}
+By default, the Observability Pipelines Worker's data directory is set to `/var/lib/observability-pipelines-worker` - if you are using the sample configuration, you should ensure that this has at least 288GB of space available for buffering.
+
+Where possible, it is recommended to have a separate SSD mounted at that location.
+{{% /tab %}}
+{{< /tabs >}}
+
+## Connect forwarders to the Worker
+After you install and configure the Observability Pipelines Worker to send logs to your Splunk index, you must update your existing collectors to point to the Worker.
+
+For most collectors, changing the URL to the load balancer provisioned by these configurations, and updating the token, should suffice.
+
+At this point, your logs should be going to the Worker and be available for processing. The next section goes through what process is included by default, and the additional options that are available.
+
+## Working with data
+The sample configurations provided do the following:
```

Comment:
```suggestion
The sample Observability Pipelines configuration does the following:
- **Collects** logs being sent from the Splunk forwarder to the Observability Pipelines Worker. 
- **Transforms** logs by adding tags to data that has come through the Observability Pipelines Worker. This helps determine what traffic still needs to be shifted over to the Worker as you update your clusters. These tags also show you how logs are being routed through the load balancer, in case there are imbalances.
- **Routes** the logs by dual-shipping the data to both Splunk and Datadog. This demonstrates how easy it is to write to multiple destinations!
```